test/E2ETest/TyeRunTests.cs

```diff
@@ -75,12 +76,27 @@ public async Task FrontendBackendRunTest()
             });
         }
 
-        [Fact]
+        [Fact(Skip = "Need to figure out how to install func before running")]
```

Comment:
A new build step in the azure-pipelines yaml?

Reply:
Yeah eventually, or something line that.